queries.yaml

```diff
@@ -177,6 +177,23 @@ post_process:
         14: { area:  100, volume: 200000 }
         15: { area:  100, volume: 100000 }
 
+  - fn: vectordatasource.transform.build_fence
+    params:
+      base_layer: landuse
+      start_zoom: 16
+      prop_transform:
+        kind: true
+        id: true
+        source: true
+        area: true
+        min_zoom: true
```

Comment:
For the feature we are testing against (zoo enclosure) this results in the amenity tag being dropped so fence only has basic props.

Sent from my handsful device.



> On Jul 15, 2016, at 07:59, Robert Marianski <notifications@github.com> wrote:
> 
> In queries.yaml:
> 
> > @@ -177,6 +177,23 @@ post_process:
> >          14: { area:  100, volume: 200000 }
> >          15: { area:  100, volume: 100000 }
> >  
> > +  - fn: vectordatasource.transform.build_fence
> > +    params:
> > +      base_layer: landuse
> > +      start_zoom: 16
> > +      prop_transform:
> > +        kind: true
> > +        id: true
> > +        source: true
> > +        area: true
> > +        min_zoom: true
> Is there a difference between this list of properties and the source feature? If not, would it be better to just make a copy of all the source properties instead of specifying them here? That way if we add a property to landuse we don't have to remember to add it in here.
> 
> ―
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>